### ISSUE TITLE
Image classification transfer learning workflow

### ISSUE THREAD

**Issue by EmbraceLife** [EmbraceLife/shendusuipian#108]
### Image classification transfer learning workflow
	1. download dataset
	2. create DataBunch
	3. see your data
	4. create CNN with pretrained model
	5. fit the model with one-cycle-learning
	6. see how your model doing
	7. tweak and improve the workflow above
		* choose the appropriate epochs number
		* training the full model with appropriate learning rate
		* use more layers
		* data augmentation
目录 fastai DL workflow #63
